install.sh

```diff
@@ -373,7 +363,7 @@ __title() {
 "
  _ \ _ _| __ __|  \\
    /   |     |   _ \\
-_|_\ ___|   _| _/  _\\
+_|_\ ___|   _| _/  _\\  $_RITA_VERSION
```

Comment:
Version string should be printed out in the `__title` somewhere so that it is visible no matter how the installer is run.